h2o-core/src/main/java/water/fvec/Chunk.java

```diff
@@ -793,4 +793,105 @@ public final void reportBrokenCategorical(int i, int j, long l, int[] cmap, int
       sb.append("at8_abs[" + (k+_start) + "] = " + atd(k) + ", _chk2 = " + (_chk2 != null?_chk2.atd(k):"") + "\n");
     throw new RuntimeException(sb.toString());
   }
+
+  /**
+   * Generic abstraction over Chunk setter methods.
+   */
+  public interface ValueSetter {
```

Comment:
Actually, there is (currently) setAny method, but I plan to kick it out, trying to introduced TypedVectors etc. This one, it looks a little bit too heavy to me. And more, how come there's a setter, but not a getter; and how come it's not typed? Looks weird a little bit. Just musing, not suggesting anything.